### PR TITLE
Add more incorrect words

### PR DESCRIPTION
**Pull request by isaak654**

### CODE REVIEW

**Review by luzpaz — APPROVED** [review]

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -19252,12 +19260,14 @@ mcroscopes->microscopes
...
 mcroscopies->microscopies
 mcroscopy->microscopy
+mdoe->mode
 mdoel->model
 mdoeled->modeled
 mdoeling->modeling
 mdoelled->modelled
 mdoelling->modelling
 mdoels->models
+mdoes->modes
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:19270]: Suggest does too.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -19252,12 +19260,14 @@ mcroscopes->microscopes
 mcroscopic->microscopic
 mcroscopies->microscopies
 mcroscopy->microscopy
+mdoe->mode
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:19263]: Suggest doe (female deer) too

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -18863,6 +18869,8 @@ logictech->logitech
 logile->logfile
 loging->logging, lodging,
 logitude->longitude
+logitudes->longitudes
+logner->longer
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:18873]: Suggest logger too

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -8363,6 +8364,8 @@ createdd->created
 createing->creating
 createive->creative
 creater->creator
+creaters->creators
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:8367]: Suggest craters too

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -8363,6 +8364,8 @@ createdd->created
...
+creatin->creation, creating, creatine,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:8368]: Suggest cretin too

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -5199,6 +5199,7 @@ centruy->century
 centuties->centuries
 centuty->century
 cerate->create
+cerated->created
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:5202]: Suggest serrated too

**Review by peternewman — CHANGES_REQUESTED** [review]
A few improvements please.

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done.

**Review by peternewman — APPROVED** [review]
LGTM, thanks for the changes!

**Comment by luzpaz** [comment]
LGTM

**Review by luzpaz — APPROVED** [review]

**Pull request merged**